helix-core/src/main/java/org/apache/helix/monitoring/mbeans/MonitorDomainNames.java

```diff
@@ -28,5 +28,6 @@
   HelixThreadPoolExecutor,
   HelixCallback,
   RoutingTableProvider,
-  CLMParticipantReport
+  CLMParticipantReport,
+  WagedRebalancer
```

Comment:
Let's just call it Rebalancer? Maybe we want to fill other metrics regarding the legacy rebalancers as well?
I'm not sure about this, more like a question.

Reply:
let's start with this and change it later on :)

Reply:
No, the problem is that, once you have the name, changing it requires us to consider backward compatibility. Better to do it right on the first shot.